tests/unit/modules/network/ios/test_ios_acls.py

```diff
@@ -2303,3 +2303,59 @@ def test_ios_acls_overridden_remarks_complex(self):
             "no ip access-list extended test_acl",
         ]
         self.assertEqual(sorted(result["commands"]), sorted(commands))
+
+    def test_ios_acls_with_arp_acl(self):
+        self.execute_show_command.return_value = dedent(
+            """\
+            ip access-list standard 10
+            10 permit 192.168.1.0 0.0.0.255
+            arp access-list arp-test
```

Comment:
Can we please add a integration test where we add a prepare test to add arp access-list and then gather facts so that we are sure with an appliance too that the facts are handled correctly?

Reply:
Sure will do it thanks